### ISSUE TITLE
1.26.0 - error when opening ssh:// - "Error handling readFile for: /pyproject.toml"

### ISSUE THREAD

**Issue by christianl3chn3r** [SchoofsKelvin/vscode-sshfs#379]
I'm getting this error when opening ssh:// with VSCode. This happens with any host in my configured list.
VS Code version: 1.77.2
SSH FS version: 1.26
MacOS version: 13.3.1

<img width="1110" alt="Screenshot 2023-04-11 at 11 32 04 AM" src="https://user-images.githubusercontent.com/74074420/231257692-7f187563-a1e7-4cee-ba49-ce74eac3c9b9.png">

**Comment by james-at-klik** [comment]
I can report the same issue.
Never worked with Python before, not sure what this is!
VS Code: 1.77.3
SSH FS: v1.26.0
MacOS: 11.4

**Comment by kfhunter** [comment]
Same error opening workspaces with SSH connections. The explorer tab just shows the connection or folder with the spinny never loading. I don't use any Python packages.
VSC 1.77.3
SSH FS 1.26.0
Windows 11

**Comment by David-COUDRAY** [comment]
Same error since a few weeks.
I don't use Python.

**Comment by banq** [comment]
me too ,fuck python

**Comment by SchoofsKelvin** [comment]
Took me a while to hunt down the root cause, but found it. From VS Code 1.76 and 1.77, their `product.json` file (present in the built application but not on the repository) got a new `configBasedExtensionTips` entry:

```json
"python-formatter": {
	"configPath": "pyproject.toml",
	"configName": "Python Formatter",
	"recommendations": {
		"ms-python.black-formatter": {
			"name": "Black Formatter",
			"contentPattern": "(^\\s*\\[\\s*\"?tool\"?\\s*\\.\\s*\"?black\"?\\s*\\])|(\"black\\s*[\"[(<=>!~;@])"
		}
	}
}
```

So VS Code itself (not an extension, not even a built-in one) tries to read the `pyproject.toml` file without `stat`'ing it to check for its existence, which often doesn't exist hence the error. This (usually occurring after opening the Extensions view) is to suggest extension recommendations based on certain factors. This should've been ignored by the extension (only printing a small entry in the `SSH FS` output) but a mistake while fixing #282 enabled the notification by default for read errors too instead of just write errors.

I've just pushed a commit that fixes this (d878b78) which will be available in the next version of the extension (v1.26.1) which should be released later today.
